pyre/extra_tests/snippets/stdlib_warnings.py

```
@@ -112,20 +113,61 @@ def get_source(self, fullname):
         return "first line\nsecond line\n"
 
 
+class Spec:
+    def __init__(self, loader):
```

Ruff (0.16.3)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)